modules/exploits/windows/misc/cisco_webex_ext.rb

```diff
@@ -0,0 +1,130 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::HttpServer
+  include Msf::Exploit::EXE
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Cisco WebEx Chrome Extension RCE (CVE-2017-3823)",
+      'Description'    => %q{
+        This module exploits a vulnerability present in the Cisco WebEx Chrome Extension
+        version 1.0.1 which allows an attacker to execute arbitrary commands on a system.
+        },
+        'License'        => MSF_LICENSE,
+        'Author'         =>
+        [
+          'Tavis Ormandy <taviso@google.com>',                # Original research/PoC
+          'William Webb <william_webb[at]rapid7.com>'         # Metasploit module
+        ],
+        'Platform'       => 'win',
+        'DefaultOptions' =>
+        {
+          'SSL' => true,
+        },
+        'Targets'        =>
+        [
+          [ 'Cisco WebEx Extension 1.0.1',
+            {
+              'Platform' => 'win',
+              'Arch'     => ARCH_X86,
+            }
+          ],
+        ],
+        'References'     =>
+        [
+          [ 'CVE', '2017-3823' ],
+        ],
+        'Arch'           => ARCH_X86,
+        'DisclosureDate' => "Jan 21 2017",
+        'DefaultTarget'  => 0
```

Comment:
You can stick `'SSL' => true` in `DefaultOptions` after this. At least you don't have to use `HttpClient`, too. :P